src/components/Sum.jsx

```diff
@@ -0,0 +1 @@
+export const Sum = ({ a = 0, b = 0 }) => `Sum of ${a} and ${b} is ${a + b}`;
```

Comment:
You need to export React component with JSX, not simple string.
Also, need to add props validation.